core/types/transaction_signing.go

```diff
@@ -112,6 +112,9 @@ type EIP155Signer struct {
 }
 
 func NewEIP155Signer(chainId *big.Int) EIP155Signer {
+	if chainId == nil {
+		chainId = new(big.Int)
+	}
```

Comment:
types.MakeSigner is used in many places without prior BlockChain initialisation. If you look at the diff, you'll see that this default assignment was previously done in cmd/geth.